packages/gatsby-source-wordpress/src/steps/declare-plugin-options-schema.js

```diff
@@ -682,8 +686,21 @@ When using this option, be sure to gitignore the wordpress-cache directory in th
               },
             `),
         }),
+      generateAvifImages: Joi.boolean()
+        .default(hasImageCDN)
```

Comment:
The options schema here is used to generate the options docs page. If you run `yarn build` it should generate it.